packages/table/preview/index.tsx

```diff
@@ -468,6 +528,31 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     // State updates
     // =============
 
+    // designed to be called from componentWillMount and componentWillUpdate, hence it expects nextProps
+    private syncCellContent = (nextState?: IMutableTableState) => {
```

Comment:
That would cause the equality checks below to `return` on `componentWillMount` (because the default value would cause `isNextStateDefined` to never be `false`); I do want to go through all the content generation at that time, so I'll presumably need to know if the real `nextState` is defined or not.